testing/no-std-tests/Cargo.toml

```diff
@@ -7,7 +7,9 @@ resolver = "2"
 
 [dependencies]
 subxt-metadata = { path = "../../metadata", default-features = false }
-subxt-signer = { path = "../../signer", default-features = false, features = ["sr25519"] }
+subxt-core = { path = "../../core", default-features = false }
+subxt-signer = { path = "../../signer", default-features = false, features = ["subxt", "sr25519"] }
+subxt-macro = { path = "../../macro" }                                                                         # Not used yet, but will be used here very soon, to verify the generated code is applicable.
```

Comment:
I'm pretty slow and that follow the comment :package: 

I suppose you mean that we don't test that the generated code is no-std?